app/usecase/url/urlretriever_fake_test.go

```diff
@@ -0,0 +1,149 @@
+package url
+
+import (
+	"short/app/entity"
+	"testing"
+	"time"
+
+	"github.com/byliuyang/app/mdtest"
+)
+
+type fakeUrlMap = map[string]entity.URL
+
+func TestFakeUrlRetriever_GetUrlAfter(t *testing.T) {
+	now := time.Now()
+	before := now.Add(-5 * time.Second)
```

Comment:
```suggestion
	before := now.Add(time.Second)
```

Reply:
You ideally want to subtract right? Why are we removing the negation?